templates/webapps/predictions.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'default.html' %}
+
+{% block custom_meta %}
+  <meta name="google-signin-client_id" content="{{ CLIENT_ID }}">
+  <meta name="google-signin-scope" content="profile email">
+  <meta name="google-signin-hosted_domain" content="college.harvard.edu">
+{% endblock %}
+
+
+{% block content %}
+
+<div class="container">
+
+  <div class="intro">
+    <h1>What will happen this semester?</h1>
+    <p>Tell us how likely you think the following events are to occur. HODP will score your predictions according to <a href="https://fivethirtyeight.com/features/how-to-play-our-nfl-predictions-game/">these rules</a>, and participants with the highest scores will receive prizes at the end of the semester! You'll be able to change your predictions until Jan. 31st at midnight. (A big thank you to FiveThirtyEight, whose annual <a href="https://projects.fivethirtyeight.com/2019-nfl-forecasting-game/">NFL forecasting game</a> was the inspiration for this project.)</p>
+    {% if not signed_in %}
+      <p><a href="/auth/predictions">Sign in</a> to play!</p>
+    {% endif %}
+    {% if responded %}
+      <h5>Your predictions have been saved!</h5>
+    {% endif %}
+  </div>
+
+  <form method = "POST">
+
+    {% for question in questions %}
+      <div class="question">
+        <h4 class="question-text">{{ question.text }}</h4>
+        <p>Your prediction: <span class="prediction" id="q{{ question.number }}-slider-value">50%</span> chance that {{ question.outcome_1 }}</p>
+        <div class="row row{{ question.number }}">
+          <div class="col-md-6 col1">
+            <input type="range" name="{{ question.name }}" min="0" max="100" class="slider" id="q{{ question.number }}-slider"
+            {% if predictions is defined and question.name in predictions %}
+              value = {{ predictions[question.name] }}
+            {% endif %}>
+          </div>
+          <div class="col-md-6 col2">
+            <p>You'll <span class="points" id="q{{ question.number }}-1-point-change">gain 0</span> points if {{ question.outcome_1 }}.</p>
+            <p>You'll <span class="points" id="q{{ question.number }}-0-point-change">gain 0</span> points if {{ question.outcome_0 }}.</p>
+          </div>
+        </div>
+      </div>
+    {% endfor %}
+
+    {% if signed_in %}
+      <div id="end-buttons">
+        <input type="submit" class="button" id="submit" value="Save your predictions"></button>

```

Comment:
Why do we have a class called button. Can we call it like "end-button" or something